commands/displayers/apps.go

```diff
@@ -284,3 +284,75 @@ func (is AppInstanceSizes) JSON(w io.Writer) error {
 	e.SetIndent("", "  ")
 	return e.Encode(is)
 }
+
+type AppProposeResponse struct {
+	Res *godo.AppProposeResponse
+}
+
+var _ Displayable = (*AppProposeResponse)(nil)
+
+func (r AppProposeResponse) Cols() []string {
+	cols := []string{
+		"AppNameAvailable",
+	}
+
+	if r.Res.AppNameSuggestion != "" {
+		cols = append(cols, "AppNameSuggestion")
+	}
+
+	cols = append(cols, []string{
+		"AppIsStatic",
+		"StaticSites",
+		"AppCost",
+		"AppTierUpgradeCost",
+		"AppTierDowngradeCost",
+	}...)
+
+	return cols
+}
+
+func (r AppProposeResponse) ColMap() map[string]string {
+	return map[string]string{
+		"AppNameAvailable":     "App Name Available?",
+		"AppNameSuggestion":    "Suggested App Name",
+		"AppIsStatic":          "Is Static?",
+		"StaticSites":          "Free Static Site Usage",
+		"AppCost":              "$/month",
+		"AppTierUpgradeCost":   "$/month on higher tier",
+		"AppTierDowngradeCost": "$/month on lower tier",
+	}
+}
+
+func (r AppProposeResponse) KV() []map[string]interface{} {
+	staticSites := fmt.Sprintf("%s of %s", r.Res.ExistingStaticApps, r.Res.MaxFreeStaticApps)
```

Comment:
Looks like the API does not return the `existing_static_apps` field at all if no static sites are in use. Since it is a string, not an int, the default value is blank rather than `0`. So you end up with:

```
Free Static Site Usage
of 3 
```

You could handle it here, but you might want to fix that on the API side.

Reply:
Good catch @andrewsomething! I updated the representation of free static apps